### DISCUSSION TITLE
Map - initial Koordinaten

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1644] [Q&A]
Wie bekomme ich es hin, dass das Widget Map als Initialen Koordinaten die Position nimmt, die vom Modul übergeben wird?

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/system-effektprog./v3-systemprogramme/setdefaultposition
